drizzlepac/haputils/pipeline_graphics.py

```diff
@@ -0,0 +1,439 @@
+                           'gaia_fit_residuals.y',
+                           'gaia_fit_residuals.ref_x',
+                           'gaia_fit_residuals.ref_y']
+
+
+    fit_data = get_pandas_data(pandas_file, results_columns)
+    resids_data = get_pandas_data(pandas_file, resids_columns)
+    gaia_fit_data = get_pandas_data(pandas_file, gaia_fit_columns)
+    gaia_resids_data = get_pandas_data(pandas_file, gaia_resids_columns)
+    
+    fitCDS = ColumnDataSource(fit_data)
+    residsCDS = ColumnDataSource(resids_data)
+    gaiafitCDS = ColumnDataSource(gaia_fit_data)
+    gaiaresidsCDS = ColumnDataSource(gaia_resids_columns)
+    
+    gaia_output = 'gaia_{}'.format(output)
+
+    if output_dir is not None:
+        summary_filename = os.path.join(output_dir, output)
+    else:
+        summary_filename = output
+        
+    # Generate the astrometric plots
+    relative_plot_name = generate_summary_plots(fitCDS, results_columns,
+                                                  output=summary_filename)
+    
+    resids_plot_names = [relative_plot_name]
+    
+    # Generate plots for absolute alignment
+    absolute_plot_name = generate_summary_plots(fitCDS, results_columns,
+                                                base_title='Alignment to {}'.format(
+                                                  output=summary_filename)
+
+    # Generate residual plots for relative alignment
+    for filename,rootname in zip(fitCDS.data['header.FILENAME'], fitCDS.data['index']):
+        i = residsCDS.data['index'].tolist().index(rootname)
+        resids_dict = {'x': residsCDS.data[resids_columns[0]][i],
+                       'y': residsCDS.data[resids_columns[1]][i],
+                       'xr': residsCDS.data[resids_columns[2]][i],
+                       'yr': residsCDS.data[resids_columns[3]][i]}
+        cds = ColumnDataSource(resids_dict)
+
+        rms_x = float(fitCDS.data[results_columns[0]][i])
+        rms_y = float(fitCDS.data[results_columns[1]][i])
+        
+        resids_plot_name = generate_relative_residual_plots(cds, filename, 
+                                                   rms=(rms_x,rms_y),
+                                                   output_dir=output_dir,
+                                                   output=output)
+        resids_plot_names.append(resids_plot_name)
+
+    # Generate residual plots for alignment to GAIA
+    for filename,rootname in zip(gaiafitCDS.data['header.FILENAME'], gaiafitCDS.data['index']):        
+        # Now add plots for fits to GAIA
+        j = gaiaresidsCDS.data['index'].tolist().index(rootname)
+        resids_dict = {'x': gaiaresidsCDS.data[gaia_resids_columns[0]][i],
+                       'y': gaiaresidsCDS.data[gaia_resids_columns[1]][i],
+                       'xr': gaiaresidsCDS.data[gaia_resids_columns[2]][i],
+                       'yr': gaiaresidsCDS.data[gaia_resids_columns[3]][i]}
+        gaia_cds = ColumnDataSource(gaia_resids_dict)
```

Comment:
Should line 424 be 
gaia_cds = ColumnDataSource(resids_dict)???

Reply:
This was intended to 'package' up the GAIA fit specific residuals columns from 'gaia_resids_dict', not the 'resids_dict' defined previously around line 403 and 407 which contains data from the relative fit instead.

Reply:
Fixed in a7a8f70 (based on successfully generating plots with the correct data).